system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -207,7 +215,7 @@
                     {:accept-format :xml
                      :raw? true})
           {:keys [errors]} (ingest/parse-ingest-body :xml response)]
-      (is (re-find #"required key \[Name\] not found" (first errors))))))
+      (is (re-find #"You do not have permission to perform that action." (first errors))))))
```

Comment:
This test needs to be updated. It is not testing what it is supposed to test.

Reply:
ignore my comment. I think the test is just testing any error in response. Even though the error message in this case is awful, I guess it is OK.